src/aind_metadata_service/labtracks/client.py

```diff
@@ -304,3 +342,57 @@ def map_response_to_subject(self, results: List[dict]) -> List[Subject]:
             )
             subjects.append(subject)
         return subjects
+
+    @staticmethod
+    def map_response_to_procedures(
+        results: List[dict],
+    ) -> List[SubjectProcedure]:
+        """
+        Maps a response from LabTracks to an aind_data_schema.Procedure
+        Parameters
+        ----------
+        results : List[dict]
+            Results pulled from LabTracks. In the form of [row]
+
+        Returns
+        -------
+            A List of mapped Subjects (not validated)
+        """
+
+        procedures_list = []
+
+        for result in results:
+            start_date = result.get(
+                TaskSetQueryColumns.DATE_START.value
+            ).date()
+            end_date = result.get(TaskSetQueryColumns.DATE_END.value).date()
```

Comment:
Avoid chaining something after get. It will raise an exception if the return value of get is NoneType.